Tests/GRPCTests/CompressionTests.swift

```diff
@@ -24,7 +24,7 @@ class MessageCompressionTests: GRPCTestCase {
   var group: EventLoopGroup!
   var server: Server!
   var client: ClientConnection!
-  var defaultTimeout: TimeInterval = 0.1
+  var defaultTimeout: TimeInterval = 1.0
```

Comment:
Why did this need to be changed?

Reply:
One of the tests was occasionally hitting the timeout.